### ISSUE TITLE
.map files missing from the package causing warning in CRA 5.0.1 React apps

### ISSUE THREAD

**Issue by UmarFKhawaja** [acro5piano/typed-graphqlify#328]
.map files are not part of the package.

When `typed-graphqlify` is added to a React app created with `create-react-app` v5.0.1, `npm run start` shows the following warning.

```
WARNING in ../../../node_modules/typed-graphqlify/dist/index.es.js
Module Warning (from ../../../node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/mnt/Data/1/Sources/GitHub/FooBar/node_modules/typed-graphqlify/dist/index.es.js.map' file: Error: ENOENT: no such file or directory, open '/mnt/Data/1/Sources/GitHub/FooBar/node_modules/typed-graphqlify/dist/index.es.js.map'
```

**Comment by acro5piano** [comment]
Fixed by #329

**Comment by acro5piano** [comment]
Released on v3.1.5

https://github.com/acro5piano/typed-graphqlify/releases/tag/v3.1.5
